addons/website/static/src/snippets/s_accordion/options.js

```diff
@@ -0,0 +1,83 @@
+import { MediaDialog } from "@web_editor/components/media_dialog/media_dialog";
+import options from "@web_editor/js/editor/snippets.options";
+
+options.registry.Accordion = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        this.faClassActiveCustomIcons = this.$target[0].dataset.activeCustomIcon || '';
+        this.faClassInactiveCustomIcons = this.$target[0].dataset.inactiveCustomIcon || '';
+        return this._super.apply(this, arguments);
+    },
```

Comment:
"when creating the option, check the DOM to initialize some variable" -> this is wrong: this is only a cause for bugs, potentially losing the coupling between that variable and the actual DOM at some point. And I don't see the purpose, can't you just always check what the current icons are by looking in the dataset directly?

EDIT: I see your just copy the logic of the rating snippet... well it was wrong. Do not change the rating snippet but make this new one right :pray: